modules/security-context-constraints-command-reference.adoc

```diff
@@ -103,35 +104,39 @@ the default SCCs.
 //other than priority, users, groups, labels, and annotations.
 ====
 
-[id="deleting-security-context-constraints_{context}"]
-== Deleting security context constraints
+[id="updating-security-context-constraints_{context}"]
+== Updating security context constraints
 
-To delete an SCC:
+If your custom SCC no longer satisfies your application workloads requirements, you can update your SCC by using the OpenShift CLI (`oc`).
+
+To update an existing SCC:
 
 [source,terminal]
 ----
-$ oc delete scc <scc_name>
+$ oc edit scc <scc_name>
 ----
 
-[NOTE]
+[IMPORTANT]
 ====
-If you delete a default SCC, it will regenerate when you restart the cluster.
+To preserve customized SCCs during upgrades, do not edit settings on
+the default SCCs.
+//other than priority, users, groups, labels, and annotations.
 ====
 
-[id="updating-security-context-constraints_{context}"]
-== Updating security context constraints
+[id="deleting-security-context-constraints_{context}"]
+== Deleting security context constraints
 
-To update an existing SCC:
+If you no longer require your custom SCC, you can delete the SCC by using the OpenShift CLI (`oc`).
+
+To delete an SCC:
 
 [source,terminal]
 ----
-$ oc edit scc <scc_name>
+$ oc delete scc <scc_name>
 ----
 
-[NOTE]
+[IMPORTANT]
 ====
-To preserve customized SCCs during upgrades, do not edit settings on
-the default SCCs.
-//other than priority, users, groups, labels, and annotations.
+Do not delete default SCCs. If you delete a default SCC, it will regenerate when you restart the cluster.
```

Comment:
Use present tense whenever possible: "..., it regenerates when you restart the cluster."

Reply:
Thanks, this was existing text but will update!